### PR TITLE
EWPP-192: Add translation toggle label variable to file patterns.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: templates/patterns/file_teaser/pattern-file-teaser.html.twig [hunk 1]
```diff
@@ -48,7 +48,7 @@
   },
   'translation': {
     'toggle': {
-      'label': "Available translations (@count)"|t({ '@count': translations|length}),
+      'label': translation_toggle_label ~ " (@count)"|t({ '@count': translations|length}),
```
Comment (ademarco) [inline templates/patterns/file_teaser/pattern-file-teaser.html.twig]: Let's simply concatenate this string, not passing via t()
```suggestion
      'label': translation_toggle_label ~ " (" ~ translations|length ~ ")",
```

Same below

**Review by ademarco — COMMENTED** [review]
Only one comment, rest is ok, haven't tested it manually though.

File: templates/patterns/file_teaser/pattern-file-teaser.html.twig [hunk 2]
```diff
@@ -48,7 +48,7 @@
   },
   'translation': {
     'toggle': {
-      'label': "Available translations (@count)"|t({ '@count': translations|length}),
+      'label': translation_toggle_label ~ " (" ~ translations|length ~ ")",
```
Comment (ademarco) [inline templates/patterns/file_teaser/pattern-file-teaser.html.twig]: In order to make this backward compatible, i.e. when people are using patterns that don't have the new field set, let's do this:
```suggestion
      'label': translation_toggle_label|default("Available translations"|t) ~ " (" ~ translations|length ~ ")",
```

File: templates/patterns/file_translation/pattern-file-translation.html.twig [hunk 3]
```diff
@@ -45,7 +45,7 @@
   },
   'translation': {
     'toggle': {
-      'label': "Available translations (@count)"|t({ '@count': translations|length}),
+      'label': translation_toggle_label ~ " (" ~ translations|length ~ ")",
```
Comment (ademarco) [inline templates/patterns/file_translation/pattern-file-translation.html.twig]: same remark as above.

**Review by ademarco — CHANGES_REQUESTED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**